src/components/Navbar/links.tsx

```diff
@@ -10,6 +10,7 @@ export const navLinks: NavLink[] = [
   { title: "Events", slug: "events" },
   { title: "Visit Us", slug: "visit" },
   { title: "Contact Us", slug: "contact" },
+  { title: "Profile", slug: "profile"},
```

Comment:
Can you try to explain the purpose of `title` and `slug` here

Reply:
title is the text shown on the navbar itself. slug is the link of the page i.e. profile page will be accessed via /profile

Reply:
yup nice one!